src/pages/search/SearchForm.tsx

```diff
@@ -13,36 +14,24 @@ interface Props {
 const SearchForm = ({ filter, onFilterChange }: Props) => {
   return <Container>
     <Row>
-      <Col>
-        <SearchFilter label="Add Destination">
+      <ButtonGroup>
+        <SearchFilter label="Home Type" width="9rem">
+          <HomeType
+            homeType={filter.homeType}
+            onChange={homeType => onFilterChange({ ...filter, homeType })}
+          />
+        </SearchFilter>
+        <SearchFilter label="Destination" width="24rem">
```

Comment:
"Add Destination" looked inconsistent next to "Home Type" (verb versus noun); with multiple buttons, I think the context is clear enough that just "Destination" is fine here, but want to call that out for review

Reply:
i think now that we have the home type filter next to it 'Destination' seems good now!